pkg/box/box.go

```diff
@@ -90,6 +92,34 @@ type VaultConfig struct {
 	AddressCI string `yaml:"addressCI"`
 }
 
+// VersionResolvers is the configurations used to get the latest version
+type VersionResolvers struct {
```

Comment:
Might want to use `DevenvVersionResolvers` in the name, until we make this be used other places.